ibm/resource_ibm_resource_key.go

```diff
@@ -97,12 +96,96 @@ func resourceIBMResourceKey() *schema.Resource {
 				Computed:    true,
 				Description: "crn of resource key",
 			},
+
+			"guid": {
```

Comment:
All the additional attributes added in schema has to be set  in the READ method

Reply:
done